apps/api_web/lib/api_web/views/vehicle_view.ex

```diff
@@ -84,4 +85,21 @@ defmodule ApiWeb.VehicleView do
   defp backwards_compatible_attributes(attributes, _, _) do
     attributes
   end
+
+  defp encode_carriages(%{carriages: nil} = vehicle) do
+    vehicle
+  end
+
+  defp encode_carriages(vehicle) do
+    Map.put(vehicle, :carriages, Enum.map(vehicle.carriages, &encode_carriage/1))
+  end
+
+  defp encode_carriage(carraige) do
+    %{
+      label: carraige.label,
+      carriage_sequence: carraige.carriage_sequence,
```

Comment:
question: 
- if the order is already present given that this is an ordered list, does the `carriage_sequence` need to be included? 
- and if it does need to be included, couldn't the key be `sequence` since the item is already known to be a carriage?

Reply:
Normally I wouldn't want to change the API contact in a bugfix, but I suppose this is a bit of a special case, since this information wasn't available before, and wouldn't have worked if it was, so there shouldn't be any downstream consumers depending on the shape.

For my 2c, I'd probably leave `carriage_sequence` out entirely, since it's implied by the array and the API breaks from the GTFS spec in other ways (such as putting these on the `carriage` field). That said these fields to map directly to the corresponding GTFS fields, and `carriage_sequence` is actually the only field required in the spec. For context it looks like this was originally implemented in [this ticket](https://app.asana.com/0/584764604969369/1204735400214299/f) that calls for those fields.

I'll defer to you on the final shape. If you want it dropped it's easy enough to do.

Reply:
Assuming that there's some other part of the system (maybe the consist/carriage parsing?) that's ensuring the list is in `carriage_sequence` order, I'd prefer to drop it.

Reply:
The GTFS spec requires they be in order, and RTR is generating the sequence numbers based on their position in the array so we should be covered there. The field has been removed.